src/core/core.scale.js

```diff
@@ -1205,14 +1205,14 @@ var Scale = Element.extend({
 		var scaleLabelFont = helpers.options._parseFont(scaleLabel);
 		var scaleLabelPadding = helpers.options.toPadding(scaleLabel.padding);
 		var halfLineHeight = scaleLabelFont.lineHeight / 2;
-		var scaleLabelAlign = scaleLabel.labelAlign;
+		var scaleAlign = scaleLabel.align;
```

Comment:
Thanks for renaming the option. `scaleAlign` doesn't seem quite accurate for the variable name. `scaleLabelAlign` was fine or `labelAlign` if you want to keep it shorter.

Reply:
Done